ivadomed/training.py

```diff
@@ -45,6 +48,10 @@ def train(model_params, dataset_train, dataset_val, training_params, path_output
                                 training. This training state is saved everytime a new best model is saved in the log
                                 directory.
         debugging (bool): If True, extended verbosity and intermediate outputs.
+        rank (int): the rank of the training function as a process. Default value is set to -1, 
+                                indicating that Distributed Parallel Processing will not be used.
+        world_size (int): the total number of processes the will be used to run train. Default is set to 1, 
```

Comment:
I agree. Likely, the original reason was probably because those seems to be the parameters names used in Torch interface, e.g. see https://discuss.pytorch.org/t/world-size-and-rank-torch-distributed-init-process-group/57438 

We can probably at least at our cmd level, make it more interpretable and emphasize the equivalence in the code comment.